src/z3c/sqlalchemy/model.py

```diff
@@ -8,32 +8,31 @@
 
 
 """
-Optional Model support 
+Optional Model support
 """
 
 import sqlalchemy
-from zope.interface import implements
+from zope.interface import implementer
 
-from mapper import MappedClassBase
-from interfaces import IModel
+from z3c.sqlalchemy.mapper import MappedClassBase
+from z3c.sqlalchemy.interfaces import IModel
 
 __all__ = ('Model',)
 
 
+@implementer(IModel)
 class Model(dict):
     """ The Model is an optional helper class that can be passed to the
         constructor of a SQLAlchemy wrapper in order to provide hints for the mapper
         generation.
-    """        
-
-    implements(IModel)
+    """
 
     def __init__(self, *args):
         """ The constructor can be called with a series of dict. Each dict
             represents a single table and its data (see add() method).
         """
 
-        super(Model, self).__init__()
+        super().__init__()
```

Comment:
This drops Python 2 compatibility.

I'd prefer if we had at least one version working with both Python 2 and Python 3.